src/main/java/lotto/LottoApplication.java

```diff
@@ -4,19 +4,26 @@
 import lotto.view.InputView;
 import lotto.view.ResultView;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class LottoApplication {
     public static void main(String[] args) {
         int purchaseAmount = InputView.getPurchaseAmount();
+        int manualLottoCount = InputView.getManualLottoCount();
+
+        List<List<Integer>> manualLottoNumbers = new ArrayList<>();
+        if (manualLottoCount > 0) {
+            manualLottoNumbers.addAll(InputView.getManualLottoNumbers(manualLottoCount));
+        }
```

Comment:
```suggestion
        List<List<Integer>> manualLottoNumbers = InputView.getManualLottoNumbers(manualLottoCount);
```

호출하는 쪽에서 검증하는 방식보단 위처럼 `manualLottoNumbers` 리스트 생성을 포함한 전체 작업을 `InputView`에 위임하는건 어떨까요?

manualLottoCount < 0 라면 얼리리턴으로 빈리스트를 넘겨주면 될것 같아요 :)